README.md

```diff
@@ -1,57 +1,92 @@
-# Project Name
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - nodejs
+  - ms-graph
+  - msal-js
+  - msal-node
+  - azure-app-service
+  - azure-active-directory  
+  - azure-active-directory-b2c
+name: JavaScript single-page application calling Microsoft Graph with delegated permissions to manage Azure AD B2C user accounts
+urlFragment: https://github.com/Azure-Samples/ms-identity-b2c-javascript-nodejs-management
+description: "This sample demonstrates how to manage Azure AD B2C users via Microsoft Graph in JavaScript applications"
+---
+# JavaScript application calling Microsoft Graph to manage Azure AD B2C user accounts
 
-(short, 1-3 sentenced, description of the project)
+## Overview
 
-## Features
+This chapterwise tutorial demonstrates how to use the [Microsoft Graph API](https://docs.microsoft.com/graph/overview) to perform user account management operations  such as **create**, **read**, **update** and **delete** within an [Azure AD B2C](https://docs.microsoft.com/azure/active-directory-b2c/overview) directory. The code samples in this tutorial back the [Manage Azure AD B2C user accounts with Microsoft Graph](https://docs.microsoft.com/azure/active-directory-b2c/manage-user-accounts-graph-api) article on [docs.microsoft.com](https://docs.microsoft.com/).
 
-This project framework provides the following features:
+There are two modes of communication you can use when working with the **Microsoft Graph API** to manage resources in your **Azure AD B2C** tenant:
 
-* Feature 1
-* Feature 2
-* ...
+- **Interactive**: Appropriate for run-once tasks, you use an administrator account in the B2C tenant to perform the management tasks. This mode requires an **administrator** to sign-in using their credentials before calling the **Microsoft Graph API**. This is demonstrated in [Chapter 1](./Chapter1).
```

Comment:
>administrator [](start = 159, length = 13)

"global administrator", with a link to [https://docs.microsoft.com/azure/active-directory-b2c/user-overview#work-account](https://docs.microsoft.com/azure/active-directory-b2c/user-overview#work-account)